sass/includes/_alert-message.scss

```diff
@@ -25,13 +25,19 @@
         padding-left: 0.4rem;
         padding-right: 0.4rem;
         margin-right: 0.4rem;
+        font-weight: bold;
+        text-transform: uppercase;
     }
 
     &__description {
```

Comment:
Could you please have the &__description class as so:

```SASS
    &__description {
        margin-bottom: 0;
        display: inline-block;
        a {
            color: $color__link-hover;
        }
        p {
            margin-bottom: 0;
        }
    }
```

This nests the existing `<a>` class you added within the parent `&__description` block, and also adds a `margin-bottom: 0`  to the `<p>` which we weren't expecting Wagtail to render in our static version.